assets/olive-optimizer/general/jobs/olive-optimizer-built-in-component/pipeline.yml

```diff
@@ -0,0 +1,20 @@
+$schema: https://azuremlschemas.azureedge.net/latest/pipelineJob.schema.json
+type: pipeline
+display_name: olive-optimizer-built-in-component
```

Comment:
The phrase "built-in" shouldn't be used. Consider naming this olive-optimizer-pipeline instead, which is a pattern used by some other teams. Also, rename the folder containing this asset to match.